tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+- Configure the alerts to notify of.
+
+## Before you begin
+This tutorial runs best when using a `*nix` operating system.
+
+If you don't have access to a `*nix` machine you may choose to use `Google Cloud Shell` using this [quickstart](https://cloud.google.com/shell/docs/quickstart).
+### Prerequisites
+- You must have a `Google Cloud Platform` project with enabled billing.
+- You must have access to a user with `Owner` role on the project.
+- You must have `gcloud` SDK and CLI installed on your machine ([quickstart](https://cloud.google.com/sdk/docs/quickstarts)).
+- You must have a `Slack` account.
+
+**Please notice that some of the services used in this tutorial are still `beta versions` at the time of this writing and are subject to `different or no SLA`. Please consult the documentation before using this tutorial with any production critical workflow.**
+
+### TL;DR
+If you wish to skip this step-by-step tutorial - you can use the quick setup by following the [README.md](./README.md)
+
+## Setting up your project
+
+**Please remember to put your project ID where the text indicates so with: '`<YOUR_PROJECT_ID>`'**
+
+Make sure your `gcloud` SDK and CLI environment is authenticated against the requested GCP project with a user that has Owner permissions.
+
+### 1) Create a topic 
+Run the following command to create a `Google Cloud PubSub` topic:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> beta pubsub topics create gcp-alert-service
+```
+The command will take a few seconds to run. 
+Once complete your project will contain a new topic that you can see at the [Google Cloud PubSub Console](https://console.cloud.google.com/cloudpubsub/topicList).
+We will use this topic as a sink to which log exports would write our events.
+
+### 2) Create an export sink
+Run the following command to create the export sink:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> beta logging sinks create gcp_alert_service pubsub.googleapis.com/projects/<YOUR_PROJECT_ID>/topics/gcp-alert-service --log-filter "logName=projects/<YOUR_PROJECT_ID>/logs/cloudaudit.googleapis.com%2Factivity"
+```
+
+This command creates a log export that uses the specified filter to send all `Cloud Audit Logging` activity events into the topic we created before.
+
+You can see the new export sink at the [Log Exports Console](https://console.cloud.google.com/logs/exports).
+Notice the warning icon attached to it.
+
+As part of creating the export sink - a new service account was created. This service account is used to publish event into our topic.
+However this service account still lacks the permissions required to publish into `Google PubSub`.
+
+### 3) Granting service account permissions
+Run the following command:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> projects add-iam-policy-binding <YOUR_PROJECT_ID> --member=$(gcloud --project <YOUR_PROJECT_ID> --format="value(writer_identity)" beta logging sinks describe gcp_alert_service) --role='roles/pubsub.publisher'
```

Comment:
This is getting a bit long. Could you add some line breaks?

(Also, @jmdobry do we need to indent 4 characters instead of using backticks?)